Makefile

```diff
@@ -34,7 +34,7 @@ applyTemplate:
 	oc process -f $(WHAT) | oc apply -f -
 .PHONY: applyTemplate
 
-postsubmit-update: prow-services origin-release libpod prow-monitoring build-dashboards-validation-image cincinnati
```

Comment:
Let's stop adding stuff to legacy places. If you add everything somewhere under `core-services`, you won't need to put anything to Makefile.